<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPortEdit</class>
 <widget class="QDialog" name="DlgPortEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edicion redirecci√≥n puerto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QIPProtocolComboBox" name="protocolComboBox"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Puertos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>De</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Hasta</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pubico</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="fromPublicPortLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="toPublicPortLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Privado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="fromPrivatePortLineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="toPrivatePortLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="acceptButton">
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Protocolo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="ipLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>IP local</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIPProtocolComboBox</class>
   <extends>QComboBox</extends>
   <header>Utils/PortForward.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
